python/graphstorm/config/argument.py

```diff
@@ -759,16 +759,16 @@ def no_validation(self):
 
     ### control early stop ###
     @property
-    def call_to_consider_early_stop(self):
-        """ Burning period calls to start considering early stop
+    def early_stop_burnin_rounds(self):
+        """ Burnin rounds before to start early stop
```

Comment:
```suggestion
        """ Burn-in rounds before we start checking for the early stop condition.
```

Reply:
Good. I will change other doc strings similar to this line.